wordpress-mobile/WordPress-Android
# Add default views to blogging reminders

Fixes #14715

This PR adds default views for the first screen of the blogging reminder prompt. I've used the updates resources and copys.

To test:
- Clear app data and restart
- Log in
- Turn on the blogging reminders flag in Test feature configuration
- Create a new post
- Notice the bottom sheet contains the expected views

<img width="502" alt="Screenshot 2021-06-01 at 11 48 04" src="https://user-images.githubusercontent.com/1079756/120304040-b2e06000-c2cf-11eb-9931-7d914439dcbb.png">



## Regression Notes
1. Potential unintended areas of impact
None

PR submission checklist:
